pkg/apis/core/validation/validation.go

```diff
@@ -1667,6 +1667,9 @@ func ValidatePersistentVolumeSpec(pvSpec *core.PersistentVolumeSpec, pvName stri
 		allErrs = append(allErrs, field.Required(fldPath.Child("accessModes"), ""))
 	}
 	for _, mode := range pvSpec.AccessModes {
+		if utilfeature.DefaultFeatureGate.Enabled(features.ReadWriteOncePod) && mode == core.ReadWriteOncePod {
```

Comment:
adhere to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#new-enum-value-in-existing-field

Reply:
Done, added two commits at the start to introduce validation options for PVs and PVCs, and then extended them in the (currently 4th) commit to check if the feature gate is enabled / if the feature is in use during updates